source/connection-monitoring-and-pooling/connection-monitoring-and-pooling.rst

```diff
@@ -1288,9 +1338,13 @@ In addition to the common fields defined above, this message MUST contain the fo
      - If ``reason`` is ``ConnectionError``, the associated error. The type and format of this value is flexible; see the
        `logging specification <../logging/logging.rst#representing-errors-in-log-messages>`_  for details on representing errors in log messages.
 
+   * - waitedMS
+     - Int64
+     - ``ConnectionCheckOutFailedEvent.waited`` converted to milliseconds.
```

Comment:
I chose milliseconds because this unit is used when logging `CommandSucceededEvent/CommandFailedEvent.duration`. However, at least when logging `ConnectionCheckedOutEvent`/`ConnectionCheckOutFailedEvent.waited`, nanoseconds is definitely a more appropriate unit because those wait times can be quite small. Fortunately, an application can still access the value using a smaller unit by listening to `ConnectionCheckedOutEvent`/`ConnectionCheckOutFailedEvent`.

**For reviewers**: should nanoseconds be used when logging `ConnectionCheckedOutEvent`/`ConnectionCheckOutFailedEvent.waited`?